oeo.omn

```diff
@@ -1247,6 +1586,87 @@ Class: InformationArtifact
         <http://purl.obolibrary.org/obo/BFO_0000031>
     
     
+Class: InstalledHeatFromHeatPump
```

Comment:
HeatFromHeat? I guess you mean PowerFromHeat

Reply:
Heat from heatpump is okay. A heat pump has as inputs (a) low temperature heat and (b) auxillary energy for a compressor (driven usually electrically but can be also driven mechanically. Output of a heat pump is high temperature heat.

https://en.wikipedia.org/wiki/Heat_pump